app/views/open_data/show.html.erb

```diff
@@ -57,6 +57,16 @@
       <%= render partial: 'tables', object: tables %>
       <%= render partial: 'table', collection: tables %>
 
+      <h3 id="fs">Informačné zoznamy Finančnej správy <span class="label label-danger">ALFA</span></h3>
+      <p>Informačné zoznamy Finančnej správy obsahujú informácie o daňových subjektoch na Slovensku. Momentálne poskytujeme len zoznam daňových subjektov registrovaných pre DPH a zoznam daňových subjektov s výškou nadmerného odpočtu a výškou vlastnej daňovej povinnosti.</p>
+      <p class="btn-row">
+        <a href="https://s3.eu-central-1.amazonaws.com/ekosystem-slovensko-digital-dumps/fs.sql.gz?torrent" class="btn btn-default"><i class="glyphicon glyphicon-cloud-download"></i> <strong>fs.sql.gz.torrent</strong></a>
+        <a href="https://s3.eu-central-1.amazonaws.com/ekosystem-slovensko-digital-dumps/fs.sql.gz" class="btn btn-default"><i class="glyphicon glyphicon-download-alt"></i> fs.sql.gz</a>
+      </p>
+      <% tables = PgCatalog::PgTable.find_by_schema(:fs) %>
```

Comment:
Tuto podme explicitne len tie dva lebo sa inak popalime.

Reply:
ok, chceme tu tym padom vobec drzat `structure.sql`?

Reply:
structure.sql sa podla mna nepouziva na nic ine len na testy. v produkcii je to nepouzity subor.